requirements.txt

```diff
@@ -228,7 +228,7 @@ pynpm==0.1.2
     # via
     #   flask-webpackext
     #   pywebpack
-pyparsing==3.0.1
+pyparsing==2.4.7
```

Comment:
Is this intentional? It should be on 3.0.5

Reply:
Seems to have been intentional, but apparently got reverted upstream (w/o a new release yet) in the meantime: https://github.com/pypa/packaging/pull/482

Reply:
I think it's safe to merge with the downgrade and then upgrade it again once packaging released a version w/o that pin..

Reply:
OK, fair enough